corehq/apps/locations/static/locations/ko/location_types.js

```diff
@@ -161,3 +163,8 @@ ko.bindingHandlers.bind_element = {
         }
     }
 };
+
+    return {
```

Comment:
For this specific file, it'd be less verbose to make LocationSettingsViewModel an anonymous function and return that function rather than a hash. Then in location_types.html, instead of `function($, ko, location_types) {` you'd use `function($, ko, LocationSettingsViewModel) {` and instead of `new location_types.LocationSettingsViewModel(loc_types, commtrack_enabled);` you could leave it as `new LocationSettingsViewModel(loc_types, commtrack_enabled);` (I'd also likely rename this file to location_settings_view_model.js.) Thoughts?